src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -162,7 +162,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         }
 
         // To avoid issues when the submitted choices are arrays (i.e. array to string conversions),
-        // we have to ensure that all elements of the submitted choice data are strings or null.
+        // we have to ensure that all elements of the submitted choice data are NULL, strings or ints.
```

Comment:
To conform with [exception message](https://github.com/issei-m/symfony/blob/a49d79c856ebc0d6e3ffdc6074fdabb19ee7936d/src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php#L175)👼